system/MenuButton.module.scss

```diff
@@ -32,6 +32,7 @@
   color: inherit;
 
   // (@xBalbinus) These timings should line up with the color transitioning speed in ThemeManager.tsx
-  transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out, color 0.3s ease-in-out;
-  -webkit-transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out, color 0.3s ease-in-out;
+  $transition-timing: 0.3s ease-in-out;
+  transition: transform $transition-timing, opacity $transition-timing, color $transition-timing;
+  -webkit-transition: transform $transition-timing, opacity $transition-timing, color $transition-timing;
```

Comment:
u can actually put the entire line into a variable as well.

```
$transition-timing: 0.3s ease-in-out;
$transition: transform $transition-timing, opacity $transition-timing, color $transition-timing;
transition: $transition;
-webkit-transition: $transition;
```

though it just occurred to me that u don't actually have to write the webkit variant at all, because nextjs by default automatically inserts vendor prefixes during preprocessing.

code style nit, non-blocking but good to do

Reply:
Ooooh, thank you for that